lib/UserSearch/UserSearchList.js

```diff
@@ -22,35 +22,36 @@ const UserSearchList = ({
     );
   }
   return (
-    <Fragment>
+    <ul className="user-search-list h-padding-clear h-margin-clear:q">
```

Comment:
Do you think it's worth adding the `<ul>` and `<li>` items to the `<Dropdown>` and `<Dropdown.Action>` components respectively?

Reply:
So we don't have to wrap them every time we use them in the app?

Reply:
@altaudio are you suggesting that dropdowns are only ever used for lists? Because if so that isn't the case, `ActionGroup` and `Action` would only be the case where it would warrant the fixed markup IMO.

Reply:
Sure thing, thanks for the response 👍